recipe/meta.yaml

```diff
@@ -18,35 +18,49 @@ build:
     - numba = numba.misc.numba_entry:main
   script:
     - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+  missing_dso_whitelist: # [osx]
+    # optional dependency: required only when omp is chosen as the backend for
+    # the threading layer
+    - lib/libiomp5.dylib # [osx]
   ignore_run_exports:
-    # TBB is entirely optional
-    - tbb
+    # tbb-devel triggers hard dependency on tbb, this is not the case.
+    - tbb # [not (armv6l or armv7l or aarch64 or linux32)]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    # both of these are needed on osx, llvm for the headers, Intel for the lib
+    - llvm-openmp              # [osx]
+    - intel-openmp             # [osx]
```

Comment:
cc @isuruf

Reply:
I'm not sure whether this works with [our current strategy]( https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=openmp#openmp ), but will defer to others who know more.

Reply:
Only `llvm` is needed. `intel-openmp` can be removed.

Reply:
Done.

Reply:
cc @stuartarchibald (for vis)

Reply:
thanks @jakirkham, wonder if this is needed any more, I wrote this work around in years back.

Reply:
Ofc. This might also be how OpenMP is handled now in `conda-forge` has made things easier. @mingwandroid would know if the same strategy has been applied to `defaults`.